packages/perseus/src/widgets/numeric-input/numeric-input.test.ts

```diff
@@ -113,6 +114,16 @@ describe("numeric-input widget", () => {
         expect(container).toMatchSnapshot("render with format list tooltip");
     });
 
+    it("Should render tooltip using only correct answer formats", async () => {
+        // Arrange
+        const {container} = renderQuestion(correctAndWrongAnswers);
```

Comment:
Just a thought, we could write really focused unit tests for this. Something like:

``` TS
import NumericInputClass from "./numeric-input.class.tsx"`

describe("transform", () => {
  it("only uses answer forms from correct answers", () => {
    const transform = NumericInputClass.transform;
    const widgetOptions = { /* you get the idea */ };
    const renderProps = transform(widgetOptions);
    expect(renderProps).toEqual({ /* whatever */ });
  })
})
```